### ISSUE TITLE
[Warrior - Charge] Falling under the texture

### ISSUE THREAD

**Issue by Avyxis17** [chromiecraft/chromiecraft#774]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [ ] Horde

##### CONTENT PHASE:

- [ ] Generic
- [x] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

When I charged the Redridge Mystic on top of a hill in Redridge, I fell under the texture.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

It shouldn't happen.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Go to the Redridge Mountains
2. Go to the gnoll camp
3. Charge the Redridge Mystic if he stands on the little hill like on the screenshot
4. Observe

##### EXTRA NOTES:

![WoWScrnShot_060421_112201](https://user-images.githubusercontent.com/84775498/120810442-57baa180-c54b-11eb-96fd-251da6c2420e.jpg)
![WoWScrnShot_060421_112115](https://user-images.githubusercontent.com/84775498/120810477-6012dc80-c54b-11eb-8e58-b0729c6c03cb.jpg)

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/40cf7b46d221e562a20264316d660318af213690

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Azcobu** [comment]
Thanks for the report. Unfortunately, I wasn't able to reproduce this:

https://user-images.githubusercontent.com/81782124/120826315-7182d580-c599-11eb-86a6-ae4caff1f924.mp4

Were you able to get it to happen more than once? If so, perhaps you can tell me if my angle of approach was wrong or something. I tried from a few different angles but never managed to clip through.

For anyone else trying this, the mob's GUID is 28327.

**Comment by Avyxis17** [comment]
No, it only happened once to me. As for the angle of the charge that's hard to tell. The angle in the image, in which I am yet to use charge is only a hard estimate. As it was actually taken after I fell under the texture and tried again (for better understanding of where and what). I wasn't expecting it to happen the first time I charged, so naturally I didn't take screenshot of it. Only the falling part.

**Comment by Invicta1986** [comment]
I can confirm this is happening both to me and my warrior friend as well, sadly I can not give more details at this point other than we have used Charge and randomly it has made us fall through terrain with instant respawn at graveyard as result.. we are both horde if that matters !

**Comment by Azcobu** [comment]
@Invicta1986 - cheers, have copied your comment to the AC issue.

**Comment by meudeusdosceus** [comment]
I had the same problem at Ragnnaros fight today.
I got Hammer of Wrath and charged back, but i didnt land in rag feet, i fall down through the limbo.
